### ISSUE TITLE
Change Docker Hub docs to show port 3000 not port 8080 is the default

### ISSUE THREAD

**Issue by RobbieLD** [akhilrex/hammond#112]
Please update the docker hub docs to show that port 3000 not 8080 is what's being used. I just spent a while trying to work out why port 8080 wasn't working basing my docker compose on the docker hub one.

**Comment by oskarslusitis** [comment]
This bug is still preset. Just set up the Hammond for the first time myself and the same as @RobbieLD couldn't understand what I'm doing wrong. Has this project been abandoned?

**Comment by RobbieLD** [comment]
> This bug is still preset. Just set up the Hammond for the first time myself and the same as @RobbieLD couldn't understand what I'm doing wrong. Has this project been abandoned?

yeah looks like it's not been updated for a while so it's probably abandoned. I might have a look at some of the forks and see if they are being maintained.

**Comment by RobbieLD** [comment]
> This bug is still preset. Just set up the Hammond for the first time myself and the same as @RobbieLD couldn't understand what I'm doing wrong. Has this project been abandoned?

Looks like https://github.com/AlfHou/hammond/ the one current under active development.
